### DISCUSSION TITLE
Widget borders

### DISCUSSION THREAD

**Discussion by ideahit** [baffler/Transparent-Twitch-Chat-Overlay#117] [Ideas]
It would be very useful to add a button to restore widget borders after hiding them, this works with chat but not whit widgets. Now i can only close widgets from task manager (or restarting Twitch Overlay and then closing them from the X)

**Comment by baffler** [comment]
![image](https://user-images.githubusercontent.com/1980600/158533744-066f0490-a077-42b8-b6fa-560bd2cbe6fa.png)

Are you using this to toggle the borders? It will toggle all the borders even for widgets.
